README.md

```diff
@@ -11,4 +11,55 @@ Supported formats:
    ```
     gradlew clean build
    ``` 
-3. Find the distribution in build/distributions.
\ No newline at end of file
+3. Find the distribution in build/distributions.
+
+## Transform Parasoft Reports
+
+### Transform Parasoft Static Analysis XML Report to SARIF Report
+1. Run XMLToSARIF script with the generated XML report:
```

Comment:
How about:
`Run the *XMLToSARIF* script in *bin* folder with Parasoft XML report:`
So "XMLToSARIF" and "bin" will be displayed with italic style.